src/content/defi/index.md

```diff
@@ -0,0 +1,245 @@
+
+### Swapping tokens {#swaps}
+
+There are lots of tokens on Ethereum. Decentralized exchanges let you trade your tokens while always having control of your assets.
+
+When you use a centralized exchange you have to deposit your assets before the trade and trust them to look after them. While your assets are deposited, they're at risk as centralized exchanges are attractive targets for hackers.
+
+### Trading and prediction markets {#trading}
+
+??? [ Not sure what to write here ]
+
+### Grow your portfolio {#investing}
+
+There are fund management products on Ethereum that will try to grow your portfolio based on a strategy of your choice. This is automatic and doesn't need a human manager taking a cut of your profits.
+
+TokenSets, a popular investment product, tokenises investment strategies and calls them sets. A set represents a portfolio with a strategy. For example, their "ETH 26 Day EMA Crossover" set will aim to accumulate as much ETH as possible by rebalancing the portfolio between ETH and USDC. You don't have to manually handle any portfolio rebalancing. As long as you have the Set token in your wallet, you can withraw the portfolio as individual tokens whenever you like.
+
+### Fund your ideas {#crowdfunding}
+
+Ethereum is open source software. A lot of the work so far has been funded by the community and this has led to the growth of an interesting new fundraising model: quadratic funding. This method has the potential to improve the way we fund all types of public goods in the future.
+
+Quadratic funding makes sure that the projects that receive the most funding are those with the most unique demand. In other words, projects that stand to improve the lives of the most people. Here's how it works:
+
+1. There is a matching pool of funds donated.
+2. A round of public funding starts.
+3. People can signal their demand for a project by donating some money.
+4. Once the round is over, the matching pool is distributed to projects. Those with the most unique demand get the highest amount from the matching pool.
+
+This means project A that received 100 donations of 1 dollar could end up with more funding than project B which received a single donation of 10,000 dollars (dependent on the size of the matching pool).
+
+## How does defi work?
+
+Defi uses cryptocurrencies and smart contracts to provide services that don't need intermediaries. In today's financial world, financial institutions acts as guarantors of transactions. This gives these institutions immense power – even the power to stop you making transactions because your money flows through them. And discrimination is rife – billions of people around the world don't have access to a bank account.
```

Comment:
```suggestion
Defi uses cryptocurrencies and smart contracts to provide services that don't need intermediaries. In today's financial world, financial institutions act as guarantors of transactions. This gives these institutions immense power – even the power to stop you making transactions because your money flows through them. And discrimination is rife – billions of people around the world don't have access to a bank account.
```

Reply:
Is discrimination the reason billions don't have access to a bank account?